tiled/authenticators.py

```diff
@@ -107,6 +110,87 @@ async def authenticate(self, username: str, password: str) -> UserSessionState:
         else:
             return UserSessionState(username, {})
 
+class MinimalOIDCAuthenticator:
```

Comment:
I think that the approach used by the existing `OIDCAuthenticator` is simply wrong, and it probably makes sense to replace it with this approach rather than add a second authenticator.

Reply:
I agree with this statement.

Reply:
I only didn't because it changed the required configuration values